learn/wavemaker-release-notes/v10-12-2.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "WaveMaker 10.12.2 - Release date: 14 February 2022"
+id: ""
+sidebar_label: "Release v10.12.2"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+---
+
+## Bug Fixes
+
+- Fixed an issue related to entering Time field data into the Data Table.
```

Comment:
Fixed an issue related to entering Time field data into the Data Table while performing CRUD operations on the data table bound to a DB CRUD variable.